microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -27,9 +27,9 @@ This section describes the types of verification, what is verified, and suggesti
 
 Email ownership verifies that the primary contact (primary email) address is valid.
 
-* The primary contact email address must be a work account that is monitored and can send/receive email.
-* Avoid using a personal email address not associated with the company domain, such as abc@gmail.com.
-* Avoid using a tenant user credential that's not associated with email, such asjsmith@testcompany on microsoft.com.
+* The primary contact email address must be a work account that is monitored and can send and receive email.
+* Avoid using a personal email address not associated with the company domain, such as \<alias\>@contoso.com.
+* Avoid using a tenant user credential that's not associated with email, such \<alias\>@contoso on microsoft.com.
```

Comment:
@sonakshimore  - pls review the format of this tenant user credential. 

Rendered: 
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program?branch=pr-en-us-2016#verify-email-ownership

Reply:
@mikehoffms Looks good to me! We can go ahead with this.

CC: @ashishnirvana1